meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -31,3 +31,10 @@ python __anonymous() {
 }
 
 inherit ${LMPSTAGING_INHERIT_KERNEL_MODSIGN}
+
+BB_HASHCHECK_FUNCTION:lmp = "lmp_sstate_checkhashes"
```

Comment:
Is there any reason why not to use it also for lmp-base/wayland?

Reply:
I think `lmp` override is used in all distros, if not I need to fix it.

Reply:
yup, lmp is in all distro variants
```
meta-lmp-base/conf/distro/lmp-base-xwayland.conf:DISTROOVERRIDES = "lmp:lmp-base:lmp-wayland:lmp-xwayland:lmp-base-wayland:lmp-base-xwayland"
meta-lmp-base/conf/distro/lmp-base.conf:DISTROOVERRIDES = "lmp:lmp-base"
meta-lmp-base/conf/distro/lmp-wayland.conf:DISTROOVERRIDES = "lmp:lmp-wayland"
meta-lmp-base/conf/distro/lmp-xwayland.conf:DISTROOVERRIDES = "lmp:lmp-wayland:lmp-xwayland"
meta-lmp-base/conf/distro/lmp-base-wayland.conf:DISTROOVERRIDES = "lmp:lmp-base:lmp-wayland:lmp-base-wayland"
meta-lmp-base/conf/distro/lmp-mfgtool.conf:DISTROOVERRIDES = "lmp:lmp-mfgtool"

```